content/lang/fr/index.md

```diff
@@ -0,0 +1,233 @@
+
+    <p>
+        Preact fait partie intégrante de la la plateforme web. Elle fait le différenciel entre le DOM Virtuel et le DOM réel, enregistre de vrais gestionnaires d'évènements, et s'intègre aussi parfaitement avec les autres bibliothèques.
+    </p>
+</section>
+
+
+<section class="home-section">
+    <img src="/assets/home/size.svg" alt="size">
+
+    <h2>Taille Réduite</h2>
+    
+    <p>
+        La plupart des frameworks UI sont tellement gros qu'ils comptent pour presque la totalité de la taille d'una application JavaScript.
+        Preact est différent: sa taille se fait oubliée au profit de <em>votre code</em> qui représente la majeure partie de votre application.
+    </p>
+    
+    <p>
+        Cela signifie moins de JavaScript à télécharger, parser et exécuter - laissant plus de temps à votre code, vous permettant de concevoir l'expérience que vous souhaitez sans vous battre avec un framework dont vous n'avez pas le contrôle.
+    </p>
+</section>
+
+
+<section class="home-section">
+    <img src="/assets/home/performance.svg" alt="performance">
+
+    <h2>Performance Accrue</h2>
+    
+    <p>
+        Preact est rapide, et pas seulement à cause de sa taille. C'est l'une des bibliothèques de DOM Virtuel les plus rapides du marché, grâce à une implémentation du différenciel simple et prédictible.
+    </p>
+    
+    <p>
+        Elle inclut même quelques améliorations de performances en plus comme la mise à jour en lots personnalisable, un rendu asynchrône optionnel, la réutilisation du DOM, et une gestion des évènements optimisée via [Linked State](/guide/linked-state).
+    </p>
+</section>
+
+
+<section class="home-section">
+    <img src="/assets/home/portable.svg" alt="portable">
+
+    <h2>Portable &amp; Embarquable</h2>
+    
+    <p>
+        La petite taille de Preact permet d'embarquer le paradigme du Composant DOM Virtuel dans des environnements où c'était jusque là impossible.
+    </p>
+    
+    <p>
+        Utilisez Preact pour construire les éléments composants votre application sans intégration complexe. Embarquez Preact dans un widget et utilisez les mêmes outils et techniques que vous utiliseriez pour construire une application complète.
+    </p>
+</section>
+
+
+<section class="home-section">
+    <img src="/assets/home/productive.svg" alt="productive">
+
+    <h2>Une Productivité Instantanée</h2>
+    
+    <p>
+        Ne sacrifiez plus votre productivité pour concevoir une application légère. Preact vous permet d'être productif dès le début et apporte même quelques fonctionnalités bonus:
+    </p>
+    
+    <ul>
+        <li>`props`, `state` et `context` sont passées à `render()`</li>
+        <li>Utilisez les attributs HTML standards comme `class` et `for`</li>
+        <li>Fonctionne avec React DevTools sans aucune modification</li>
+    </ul>
+</section>
+
+
+<section class="home-section">
+    <img src="/assets/home/compatible.svg" alt="compatible">
+
+    <h2>Un Ecosystème Compatible</h2>
```

Comment:
Sounds stupid, but I don't know how to type a captial letter with an accent on Windows... I copied it in the meantime :)